cmd/template/cluster/flag.go

```diff
@@ -75,6 +80,14 @@ func (f *flag) Init(cmd *cobra.Command) {
 
 	// TODO: Remove around December 2021
 	_ = cmd.Flags().MarkDeprecated(flagClusterIDDeprecated, "please use --name instead.")
+
+	f.config = genericclioptions.NewConfigFlags(true)
+	f.print = genericclioptions.NewPrintFlags("")
```

Comment:
Turns out overriding print flags is not as straight forward as overriding config flags. This works

```suggestion
	f.print = genericclioptions.NewPrintFlags("")
	f.print.OutputFormat = nil
```